src/Utils/StyleAttributesUtils.php

```diff
@@ -154,6 +154,59 @@ public static function get_background_color_class_and_style( $attributes ) {
 		return null;
 	}
 
+	/**
+	 * Get class and style for align from attributes.
+	 *
+	 * @param array $attributes Block attributes.
+	 *
+	 * @return (array | null)
+	 */
+	public static function get_align_class_and_style( $attributes ) {
```

Comment:
My idea for this class is to return always an array with class and style because I would really to have only one function for each style. This way we can easily extend the functionality in the future. What do you think?

Reply:
Make sense!